jstl/1.2/_index.md

```diff
@@ -0,0 +1,55 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Jakarta Standard Tag Library Spec 1.2"
+date: 2019-07-18T16:10:38-04:00 
+summary: ""
+---
+Jakarta Standard Tag Library encapsulates as simple tags the core functionality common to many Web applications.
+Jakarta Standard Tag Library has support for common, structural tasks such as iteration and conditionals, tags
+for manipulating XML documents, internationalization tags, and SQL tags. It also provides a framework for integrating
+existing custom tags with Jakarta Standard Tag Library tags.
+
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.pdf) (PDF)
+* [Jakarta Standard Tag Library 1.2 Specification Document](./jstl-1.2-spec.html) (HTML)
+* [Jakarta Standard Tag Library 1.2 Javadoc](./apidocs)
```

Comment:
An entry needs to be added after this one for the tag documentation, which should be included
in the javadocs PR, perhaps as a "tagdocs" directory or something similar.

Reply:
Hi Bill, can you explain more clear about this comment?

Reply:
Hi Bill, can you make it more clear to me? where can get or generate the tag docs under "tagdocs" directory?

Reply:
Hi Bill, fixed, please help to check.